common/Kconfig

```diff
@@ -490,6 +490,14 @@ config BOOTFIRMWARE_INFO
 	help
 	  Display information about the boot firmware (version etc).
 
+config BOOTFIRMWARE_INFO_STRICT
+	bool "Display information about the boot firmware"
+	depends on BOOTFIRMWARE_INFO
+	default n
+	help
+	  Display information about the boot firmware (version etc). If the
```

Comment:
We can probably drop the first line of this extended description as well, since it depends on BOOTFIRMWARE_INFO.